Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -55,6 +56,17 @@
 PROFILING_DUMP_ROWS_LIMIT = 20
 MAX_SAMPLES = 10
 
+FileUploadParams = namedtuple(
```

Comment:
Better to update declaration of `namedtuple` to be a class with typing hints (according to @shmuel44)

Reply:
Fixed in commit a16dac6b5d1baf5bffc7d6e6544be2d1ae7dad4b